pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -2548,7 +2548,7 @@ private class LibraryReference : IDisposable
 
             public LibraryReference(Library reference)
             {
-                Reference = Reference;
+                Reference = reference;

```

Comment:
I guess this was only causing a leak, since the only function of LibraryReference was to dispose the library, making it not so surprising that no test or use case ever caught this.